src/central-server/openapi-model/src/main/resources/openapi-definition.yaml

```diff
@@ -4826,6 +4826,68 @@ components:
         - member_class
         - member_code
         - server_code
+    ManagementRequestView:
+      type: object
+      description: Management request view
+      properties:
+        id:
+          type: integer
+          description: request id
+          readOnly: true
+        origin:
+          $ref: '#/components/schemas/ManagementRequestOrigin'
+        comments:
+          type: string
+          readOnly: true
+        type:
+          type: string
+          readOnly: true
+        security_server_identifier_id:
```

Comment:
How about reusing existing ManagementRequest object, but expanding it? Plus instead of using encodedIds use clientId/serverId objects and add encodedId field to those objects for places where it is needed.